.dockerignore

```
@@ -0,0 +1,27 @@
+/node_modules
+/config.js
+/coverage
+/dist
+/tmp
+.DS_Store
+.cache
+/*.log
+/.idea
+/package-lock.json
+*.pyc
+renovate-0.0.0-semantic-release.tgz
+/test/e2e/node_modules
+.eslintcache
+junit.xml
+/test-results
+renovate-schema.json
+**/gradle-wrapper/__fixtures__/**/.gradle
+.clinic/
+
+# generated code
+**/*.generated.ts
+tools/dist
+test/datasource/nuget/_fixtures/obj
+
+# testing
+/docker-compose.yml
```

```suggestion
*
```

We don't copy any files, so ignore all (Dockerfile is excluded automatically)